docs/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
,[1173,1],[1171,1],[1170,1],[2,1],[3,1],[4,1],[5,1],[6,1],[7,1],[8,1],[9,1],[710,1],[818,1],[902,1],[794,1],[814,1],[874,1],[875,1],[876,1],[903,1],[877,1],[878,1],[879,1],[872,1],[816,1],[873,1],[800,1],[987,1],[817,1],[819,1],[820,1],[821,1],[815,1],[822,1],[727,1],[756,1],[719,1],[829,1],[721,1],[717,1],[713,1],[724,1],[830,1],[807,1],[805,1],[809,1],[804,1],[801,1],[803,1],[811,1],[838,1],[901,1],[839,1],[732,1],[749,1],[840,1],[812,1],[833,1],[813,1],[834,1],[832,1],[831,1],[835,1],[836,1],[662,1],[880,1],[726,1],[808,1],[848,1],[845,1],[844,1],[842,1],[841,1],[843,1],[846,1],[847,1],[849,1],[850,1],[859,1],[860,1],[851,1],[852,1],[853,1],[855,1],[861,1],[862,1],[659,1],[663,1],[658,1],[863,1],[864,1],[865,1],[870,1],[866,1],[867,1],[869,1],[868,1],[871,1],[856,1],[51,1],[858,1],[857,1],[934,1],[711,1],[671,1],[786,1],[788,1],[787,1],[941,1],[789,1],[791,1],[790,1],[936,1],[938,1],[937,1],[940,1],[939,1],[935,1],[942,1],[810,1],[675,1],[881,1],[944,1],[882,1],[676,1],[883,1],[983,1],[884,1],[677,1],[885,1],[984,1],[746,1],[886,1],[946,1],[797,1],[967,1],[796,1],[679,1],[887,1],[947,1],[799,1],[888,1],[960,1],[966,1],[949,1],[950,1],[951,1],[747,1],[692,1],[889,1],[953,1],[680,1],[741,1],[890,1],[954,1],[897,1],[985,1],[955,1],[956,1],[748,1],[690,1],[891,1],[958,1],[957,1],[806,1],[682,1],[681,1],[892,1],[959,1],[968,1],[742,1],[683,1],[893,1],[962,1],[961,1],[965,1],[743,1],[894,1],[963,1],[745,1],[686,1],[895,1],[964,1],[896,1],[689,1],[837,1],[906,1],[905,1],[952,1],[735,1],[795,1],[798,1],[734,1],[898,1],[733,1],[900,1],[904,1],[712,1],[899,1],[737,1],[736,1],[943,1],[932,1],[909,1],[910,1],[911,1],[912,1],[913,1],[914,1],[915,1],[916,1],[917,1],[918,1],[931,1],[919,1],[920,1],[921,1],[922,1],[923,1],[924,1],[925,1],[926,1],[927,1],[928,1],[929,1],[930,1],[792,1],[793,1],[709,1],[779,1],[758,1],[759,1],[761,1],[784,1],[773,1],[763,1],[778,1],[765,1],[764,1],[775,1],[774,1],[694,1],[785,1],[767,1],[766,1],[762,1],[776,1],[777,1],[768,1],[780,1],[769,1],[770,1],[771,1],[781,1],[730,1],[728,1],[722,1],[661,1],[750,1],[757,1],[669,1],[670,1],[718,1],[664,1],[729,1],[716,1],[665,1],[725,1],[666,1],[715,1],[668,1],[751,1],[760,1],[823,1],[672,1],[52,1],[753,1],[754,1],[673,1],[678,1],[674,1],[782,1],[772,1],[695,1],[657,1],[691,1],[660,1],[697,1],[699,1],[688,1],[824,1],[53,1],[684,1],[698,1],[708,1],[685,1],[693,1],[828,1],[696,1],[700,1],[752,1],[701,1],[705,1],[706,1],[707,1],[702,1],[744,1],[704,1],[703,1],[731,1],[687,1],[825,1],[826,1],[755,1],[827,1],[783,1],[802,1],[948,1],[908,1],[723,1],[945,1],[714,1],[667,1],[933,1],[972,1],[970,1],[973,1],[969,1],[971,1],[1048,1],[1051,1],[1052,1],[1056,1],[1044,1],[992,1],[1024,1],[1043,1],[1038,1],[1041,1],[1039,1],[1042,1],[1040,1],[1026,1],[1028,1],[1025,1],[1029,1],[1030,1],[1031,1],[1035,1],[1036,1],[1045,1],[1050,1],[1047,1],[1049,1],[1037,1],[1054,1],[1055,1],[990,1],[988,1],[991,1],[986,1],[989,1],[975,1],[974,1],[976,1],[982,1],[979,1],[977,1],[978,1],[981,1],[980,1]]},"version":"4.4.4"}
```

Comment:
This file was not supposed to be commited since it's ignored: https://github.com/mui-org/material-ui-x/blob/5f80fd03ff18e64bc270f5a31248e0e8a4a6c30b/.gitignore#L9

I'm removing it in #3139